pytest-selenium/pages/content.py

```diff
@@ -0,0 +1,45 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+from pages.base import Page
+from regions.base import Region
+
+
+class Content(Page):
+    URL_TEMPLATE = "/books/{book_slug}/pages/{page_slug}"
+
+    _main_content_locator = (
+        By.CSS_SELECTOR,
+        '#main-content > div > div > div h1[data-type="document-title"]',
+    )
+    _table_of_contents_button_locator = (By.CSS_SELECTOR, "#root > div > div > button")
```

Comment:
```suggestion
    _table_of_contents_button_locator = (By.CSS_SELECTOR, "[aria-label='Click to close the Table of Contents']")
```

Relying on the `[aria-label]` rather than a fragile selector helps test accessibility _and_ is less fragile than a `>` selector